pkg/arvo/app/file-server.hoon

```diff
@@ -287,10 +287,10 @@
       *@uv
     =/  parent  (scot %p ship.u.ota)
     =+  .^(=cass:clay %cs /[parent]/[desk.u.ota]/1/late/foo)
-    %^  end  3  3
+    %^  end  0  25
     .^(@uv %cz /[parent]/[desk.u.ota]/(scot %ud ud.cass))
   --
-  
+
```

Comment:
Hi, if you get rid of this newline, it looks good to go to me.

Reply:
That line was there before...? But alright...